frontend/src/app/soundboard/soundboard-button/soundboard-button.component.ts

```diff
@@ -9,8 +10,42 @@ import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 export class SoundboardButtonComponent {
   @Input() guildId: string;
   @Input() category?: string;
+  @Input() isLocallyPlaying: boolean = false;
+  @Output() playRemote: EventEmitter<any> = new EventEmitter();
+  @Output() playLocal: EventEmitter<any> = new EventEmitter();
+  @Output() stopLocal: EventEmitter<any> = new EventEmitter();
```

Comment:
I'm absolutely not sure, if passing these into the component is good or not.

Reply:
I think that is fine, but I would suggest changing the types a bit:

```suggestion
  @Input() isLocallyPlaying = false;
  @Output() playRemote = new EventEmitter<void>();
  @Output() playLocal = new EventEmitter<void>();
  @Output() stopLocal = new EventEmitter<void>();
```